src/app/fdctl/config/default.toml

```diff
@@ -820,6 +820,10 @@ dynamic_port_range = "8900-9000"
         # this really be configurable?
         send_buffer_size = 16384
 
+        # Additional addresses this validator needs to be able to
```

Comment:
Don't need a nested namespace `multihome` for this, just stick it under net tile same as the other options.

Can you also add a detailed comment a validator operator who doesn't know what "multihome" is could use to understand this option, and why it might need to be set?

Reply:
Maybe call it `multihome_ip_addrs` ? I believe the convention in this config is array member names are plural as well.